components/automate-ui/src/app/pages/project/rules/project-rules.component.html

```diff
@@ -51,53 +51,57 @@
               <label>
                 <span class="label">{{ getAttributeLabel() }}</span>
                 <select ngDefaultControl formControlName="attribute">
-                  <option 
+                  <option
                     *ngFor="let option of attributes[ruleForm.get('type').value.toLowerCase()]; let i = index"
                     [value]="option.key"
                   >{{ option.value }}</option>
                 </select>
               </label>
+              <chef-error
+                *ngIf="ruleForm.get('conditions').controls[i].get('attribute').hasError('required') && ruleForm.get('conditions').controls[i].get('attribute').touched"
+              >Attribute is required</chef-error>
             </chef-form-field>
 
             <chef-form-field class="attribute-field">
               <label>
                 <span class="label">Operator</span>
-                <select ngDefaultControl
-                  formControlName="operator"
-                  (change)="updateConditionValues(condition)"
-                >
-                  <option value='EQUALS'>equals</option>
-                  <option value='MEMBER_OF'>member of</option>
+                <select ngDefaultControl formControlName="operator">
+                  <option *ngFor="let operator of operators"
+                    [value]="operator.key"
+                  >{{ operator.value }}</option>
                 </select>
               </label>
+              <chef-error
+                *ngIf="ruleForm.get('conditions').controls[i].get('operator').hasError('required') && ruleForm.get('conditions').controls[i].get('operator').touched"
+              >Operator is required</chef-error>
             </chef-form-field>
-            
+
             <chef-form-field class="attribute-field">
               <label>
                 <span class="label">Value</span>
                 <chef-input ngDefaultControl
                   formControlName="values"
-                  [value]="getConditionValue(condition.controls.values.value)"
-                  (change)="updateConditionValues(condition)"
+                  [value]="condition.controls.values.value"
```

Comment:
Little change; long story. First, code originally was adjusting the actual value to be either a string or a string array based on operator (`equals` or `member of`). But it only did that if you flip-flopped the operator; did not do it if you changed the value without flipping the operator... I corrected that, and added some tests, and then realized did not want all that anyway, because the API always wants an array for `values`.